alot/defaults/alot.rc.spec

```diff
@@ -332,23 +332,26 @@ thread_focus_linewise = boolean(default=True)
         # Outgoing messages will be GPG signed by default if this is set to True.
         sign_by_default = boolean(default=False)
 
-	# Alot will try to GPG encrypt outgoing messages by default when this
-	# is set to `all` or `trusted`.  If set to `all` the message will be
-	# encrypted for all recipients for who a key is available in the key
-	# ring.  If set to `trusted` it will be encrypted to all
-	# recipients if a trusted key is available for all recipients (one
-	# where the user id for the key is signed with a trusted signature).
-	#
-	# .. note:: If the message will not be encrypted by default you can
-	#           still use the :ref:`toggleencrypt
-	#           <cmd.envelope.toggleencrypt>`, :ref:`encrypt
-	#           <cmd.envelope.encrypt>` and :ref:`unencrypt
-	#           <cmd.envelope.unencrypt>` commands to encrypt it.
-	# .. note:: The values `True` and `False` are interpreted as `all` and
-	#           `none` respectively.  They are kept for backwards
-	#           compatibility to give users a change to migrate to the new
-	#           option type.  They might become deprecated in future
-	#           versions.
+        # Alot will try to GPG encrypt outgoing messages by default when this
+        # is set to `all` or `trusted`.  If set to `all` the message will be
+        # encrypted for all recipients for who a key is available in the key
+        # ring.  If set to `trusted` it will be encrypted to all
+        # recipients if a trusted key is available for all recipients (one
+        # where the user id for the key is signed with a trusted signature).
+        #
+        # .. note:: If the message will not be encrypted by default you can
+        #           still use the :ref:`toggleencrypt
+        #           <cmd.envelope.toggleencrypt>`, :ref:`encrypt
+        #           <cmd.envelope.encrypt>` and :ref:`unencrypt
+        #           <cmd.envelope.unencrypt>` commands to encrypt it.
+        # .. deprecated:: 0.4
+        #           The values `True` and `False` are interpreted as `all` and
+        #           `none` respectively.  They are kept for backwards
+        #           compatibility to give users a change to migrate to the new
+        #           option type.  They might become deprecated in future
```

Comment:
Shouldn't this sentence be removed as well as the next sentence effectively deprecates them.